src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/IBulkContentHandler.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OrchardCore.ContentManagement.Handlers;
+
+public interface IBulkContentHandler
```

Comment:
I just realized, sorry, that this and the properties, variables, etc. should all be named _event_ handlers. "Handler" is a technical term for content handlers in OC, while "event handler" is widely used for exactly this.